x-pack/plugins/cloud_security_posture/public/components/compliance_score_bar.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiToolTip, useEuiTheme } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { calculatePostureScore } from '../../common/utils/helpers';
+import { statusColors } from '../common/constants';
+
+export const ComplianceScoreBar = ({
+  totalPassed,
+  totalFailed,
+}: {
+  totalPassed: number;
+  totalFailed: number;
+}) => {
+  const { euiTheme } = useEuiTheme();
+  const complianceScore = calculatePostureScore(totalPassed, totalFailed);
+
+  return (
+    <EuiFlexGroup
+      gutterSize="none"
+      alignItems="center"
+      justifyContent="flexEnd"
+      style={{ gap: euiTheme.size.s }}
+    >
+      <EuiFlexItem>
+        <EuiToolTip
+          content={i18n.translate('xpack.csp.complianceScoreBar.tooltipTitle', {
+            defaultMessage: '{failed} failed and {passed} passed findings',
+            values: {
+              passed: totalPassed,
+              failed: totalFailed,
```

Comment:
i think we might want these to be formatted number strings (example: `5k`)

Reply:
great point, I've already raised this to @kfirpeled as well, and at that point, it was decided to leave it without abbreviations for now. 
@kfirpeled might be worth doing a full swipe over our numbers display in one of our quick wins sessions?